docssrc/src/commandtrees.md

```diff
@@ -80,6 +80,108 @@ In this example, we have our normal `/sayhi` command using the `executes()` meth
 
 </div>
 
+### uses nested arguments to reduce indentation levels
+
+sometimes we will need such a CommandTree:
+
+<div class="multi-pre">
+
+```java,Java
+new CommandTree("example")
+    .then(new LiteralArgument("arg1")
+    .then(new StringArgument("arg2")
+        .then(new StringArgument("arg3")
+            .then(new DoubleArgument("arg4", 0)
+                .then(new StringArgument("arg5"))
+                    .executes(...)))))
+    .register();
+```
+
+```kotlin,Kotlin
+CommandTree("example")
+    .then(LiteralArgument("arg1")
+        .then(StringArgument("arg2")
+            .then(StringArgument("arg3")
+                    .then(DoubleArgument("arg4", 0)
+                        .then(StringArgument("arg5")
+                            .executes { ... })))))
+    .register()
+```
+
+```kotlin,Kotlin DSL
+commandTree("example") {
+    literalArgument("arg1") {
+        stringArgument("arg2") {
+            stringArgument("arg3") {
+                doubleArgument("arg4", 0.0) {
+                    stringArgument("arg5") {
+                        anyExecutor { ... }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+</div>
+
+well, there's too much indentation, we can use nested arguments to reduce the indentation levels:
```

Comment:
This seems like it could be two sentences?

```suggestion
Well, there's too much indentation. We can use nested arguments to reduce the indentation levels:
```